packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx

```diff
@@ -64,22 +68,29 @@ const TabWrap = ({ children, onClick, selected, tab, type }: TabWrapProps) => {
     selected && `bpk-navigation-tab-wrap--${type}-selected`,
   );
 
+  const tabProps = {
+    ...(tab.dataCy && { 'data-cy': tab.dataCy }),
+    ...(tab.dataAnalyticsName && { 'data-analytics-name': tab.dataAnalyticsName }),
```

Comment:
Props 'data-cy' and 'data-analytics-name' has been used in GC header. To order to not impact header , add optional props in Nav tabs
https://github.skyscannertools.net/skyscanner/global-components/blob/e6dbbff3baa33e1a4a380dc2b27c10912cf8397e/src/header/HeaderTab/HeaderTab.tsx#L90

Reply:
And also some project are using data-cy to verify header loaded.